main.py

```diff
@@ -205,19 +208,20 @@ def build_filename(resource_type, outpath, date, duration, site, client_provider
        (str): The generated full pathname of the output file.
   """
   extensions = { 'data': 'raw.csv', 'bigquery': 'bigquery.sql'}
-  filename_format = "{date}+{duration}_{site}_{client_provider}_{metric}-{extension}"
-
+  filename_format = "{date}+{duration}_{location_properties}_{metric}-{extension}"
+  
+  location_properties = "_".join(filter(None, [site, client_country,
```

Comment:
(optional) What do you think of just `additional_properties`? `location_properties` seems wrong to me as a category for client ISP.